json-rpc/types/src/views.rs

```diff
@@ -66,6 +67,7 @@ pub enum AccountRoleView {
         expiration_time: u64,
         compliance_key: BytesView,
         preburn_balances: Vec<AmountView>,
+        received_mints_events_key: BytesView,
```

Comment:
Nit: 
```suggestion
        received_mint_event_key: BytesView,
```
That or `received_mint_events_key` whichever makes more sense to you.